src/OVAL/probes/unix/linux/rpm-helper.c

```diff
@@ -19,11 +19,100 @@
  */
 
 #include "rpm-helper.h"
-
+#include <errno.h>
 #ifdef HAVE_CONFIG_H
 #include <config.h>
 #endif
 
+
```

Comment:
I have started to implement it for rpm probe, so it remains here.

Yes, It will be better in extra file and every probe what would need it will use it.